src/core/meshcop/dataset_manager.cpp

```diff
@@ -601,11 +601,8 @@ otError DatasetManager::SendSetRequest(const otOperationalDataset &aDataset, con
     VerifyOrExit((message = NewMeshCoPMessage(mNetif.GetCoap(), header)) != NULL, error = OT_ERROR_NO_BUFS);
 
 #if OPENTHREAD_ENABLE_COMMISSIONER && OPENTHREAD_FTD
-    bool isCommissioner;
 
-    isCommissioner = mNetif.GetCommissioner().GetState() != kCommissionerStateDisabled ? true : false;
-
-    if (isCommissioner)
+    if (mNetif.GetCommissioner().IsActive())
```

Comment:
Agree. Only active commissioner will send a MGMT_ACTIVE_SET.req message to change one or more Active Operational Dataset values in the Leader. Here, for some test case, Harness intends to assign an invalid session id to verify if Leader can identify this invalid session id or not.